github_pr.py

```diff
@@ -149,6 +149,41 @@ def _return_specific_comment_prs(all_prs, filters):
         return []
 
 
+def _check_approved_mergers_file(mergers_file, comment_users):
+    with open(mergers_file) as f:
+        approved_users = f.read().splitlines()
+    comment_approved_users = [comment_user for comment_user in comment_users if comment_user in approved_users]
+    if comment_approved_users == []:
+        raise Exception("No approved mergers were found in comments")
+    else:
+        return comment_approved_users
+
+
+def _check_owner_cannot_ship(owner, comment_users):
+    matched_comment_non_owner_users = [comment_user for comment_user in comment_users if comment_user != owner]
+    if matched_comment_non_owner_users == []:
+        raise Exception("Owner not allowed to ship their own code")
+    else:
+        return matched_comment_non_owner_users
+
+
+def _merge_pr(pr, **args):
+    if args['mergeconditions']:
+        merge_comment_users = [comment.user.login for comment in _load_issue(**args).get_comments().reversed if re.search(".*%s.*" % args['mergecomment'], comment.body)]
+        merge_conditions = {}
+        for condition in args['mergeconditions'].split(','):
+            part = condition.partition("=")
+            merge_conditions[part[0]] = part[2]
+        if merge_conditions.get('approved-mergers-file'):
```

Comment:
we wanted this to be a file, that you could store in your github repo, aka our .apostles file